src/test/skip.json

```diff
@@ -128,6 +128,32 @@
                     {"category":"rendering","file":"legends.py","cases":["legends_07"]},
                     {"category":"rendering","file":"pixeldata.py","cases":["pixeldata_0_05","pixeldata_0_07","pixeldata_0_08","pixeldata_0_09","pixeldata_0_10","pixeldata_0_11"]},
                     {"category":"rendering","file":"volume.py","cases":["volume_01","volume_02"]},
+                    {"category":"databases","file":"ANSYS.py","cases":["ansys1_2","ansys2_2","ansys3_2"]},
```

Comment:
Do you have tickets in the work for these?

Reply:
Just submitted a issue ticket listing all the skips.

Reply:
Thank you!